variety_lib/Builder.py

```diff
@@ -118,7 +118,7 @@ def connect_signals(self, callback_obj):
         connection_dict = {}
         connection_dict.update(self.glade_handler_dict)
         connection_dict.update(callback_handler_dict)
-        for item in connection_dict.items():
+        for item in list(connection_dict.items()):
```

Comment:
the wrapping list() call shouldn't be needed in cases like this when we are directly iterating over the items(), 2to3 seems to add these regardless of the type of usage (several more below).